src/test/java/org/skriptlang/skript/test/tests/lang/ConditionalTest.java

```diff
@@ -0,0 +1,368 @@
+package org.skriptlang.skript.test.tests.lang;
+
+import ch.njol.skript.lang.util.ContextlessEvent;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.skriptlang.skript.lang.condition.CompoundConditional;
+import org.skriptlang.skript.lang.condition.Conditional;
+
+public class ConditionalTest {
+
+	TestConditional condTrue;
+	TestConditional condFalse;
+	TestConditional condUnknown;
+	Event event;
+
+	@Before
+	public void setup() {
+		condTrue = new TestConditional(Kleenean.TRUE);
+		condFalse = new TestConditional(Kleenean.FALSE);
+		condUnknown = new TestConditional(Kleenean.UNKNOWN);
+		event = ContextlessEvent.get();
```

Comment:
Do we have to use an Event here? Couldn't TestConditional use some other Context object?

Reply:
Good point